### PR TITLE
Making page About us and fixing color themes.

### PR DESCRIPTION
**Pull request by glebdronchenko**

### CODE REVIEW

File: src/components/About/About.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        This project was created to enable to people to monitor the weathet around the world. Also
+        it was training project. If you have any suggestions for the service work, you can contact
+        us using link bellow.
+      </p>
+      <RouterLink className={classes.link} to="/contacts">
+        Contacts
+      </RouterLink>
+      <p className={classes.textParagraph}>
+        This application works together with external forecast API service named "Foreca wether
+        API". You can read more information on{' '}
```
Comment (JaneEllison) [inline src/components/About/About.jsx]: {' '} unnecessary symbols

Comment (glebdronchenko) [reply src/components/About/About.jsx]: I have fixed it. Thanks a lot!

File: src/components/About/About.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import Link from '../Link/Link';
+
+function About() {
+  return (
+    <div className={classes.aboutContainer}>
+      <h1>About us</h1>
+      <p className={classes.textParagraph}>
+        This project was created to enable to people to monitor the weathet around the world. Also
+        it was training project. If you have any suggestions for the service work, you can contact
+        us using link bellow.
```
Comment (anton-vlasik) [inline src/components/About/About.jsx]: below

File: src/components/About/About.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { Link as RouterLink } from 'react-router-dom';
+import Link from '../Link/Link';
+
+function About() {
+  return (
+    <div className={classes.aboutContainer}>
+      <h1>About us</h1>
+      <p className={classes.textParagraph}>
+        This project was created to enable to people to monitor the weathet around the world. Also
+        it was training project. If you have any suggestions for the service work, you can contact
```
Comment (anton-vlasik) [inline src/components/About/About.jsx]: a training project

File: src/components/About/About.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import classes from './About.module.scss';
+import { Link as RouterLink } from 'react-router-dom';
+import Link from '../Link/Link';
+
+function About() {
+  return (
+    <div className={classes.aboutContainer}>
+      <h1>About us</h1>
+      <p className={classes.textParagraph}>
+        This project was created to enable to people to monitor the weathet around the world. Also
```
Comment (anton-vlasik) [inline src/components/About/About.jsx]: to let people monitoring

File: src/components/About/About.jsx (hunk 2 above)
Comment (glebdronchenko) [reply src/components/About/About.jsx]: I have changed. Thanks a lot!

File: src/components/About/About.jsx (hunk 3 above)
Comment (glebdronchenko) [reply src/components/About/About.jsx]: I have changed. Thanks a lot!

File: src/components/About/About.jsx (hunk 4 above)
Comment (glebdronchenko) [reply src/components/About/About.jsx]: I have changed. Thanks a lot!

**Review by anton-vlasik — APPROVED** [review]

**Pull request merged**